### PR TITLE
[FIX] docker/odoo/12.0: changes from 14.0 

### PR DESCRIPTION
**Pull request by JevinD**
I left out Azure attachment and libraries/packages since we wont need it on the current projects in 12.0

### CODE REVIEW

File: odoo/12.0/Dockerfile [hunk 1]
```diff
@@ -48,29 +52,31 @@ RUN curl -o odoo.deb -sSL http://nightly.odoo.com/${ODOO_VERSION}/nightly/deb/od
...
 
 # Copy Odoo addons
-COPY src/odoo-cloud-platform/attachment_s3 /addons/attachment_s3
-COPY src/odoo-cloud-platform/base_attachment_object_storage /addons/base_attachment_object_storage
-COPY src/odoo-cloud-platform/monitoring_status /addons/monitoring_status
-COPY src/odoo-cloud-platform/session_redis /addons/session_redis
+COPY src/odoo-cloud-platform/attachment_s3 odoo/addons/attachment_s3
+COPY src/odoo-cloud-platform/base_attachment_object_storage odoo/addons/base_attachment_object_storage
+COPY src/odoo-cloud-platform/monitoring_status odoo/addons/monitoring_status
+COPY src/odoo-cloud-platform/session_redis odoo/addons/session_redis
```
Comment (max3903) [inline odoo/12.0/Dockerfile:67]:
```suggestion
COPY src/odoo-cloud-platform/attachment_s3 /odoo/addons/attachment_s3
COPY src/odoo-cloud-platform/base_attachment_object_storage /odoo/addons/base_attachment_object_storage
COPY src/odoo-cloud-platform/monitoring_status /odoo/addons/monitoring_status
COPY src/odoo-cloud-platform/session_redis /odoo/addons/session_redis
```

File: odoo/12.0/templates/odoo.conf.tmpl [hunk 2]
```diff
@@ -53,6 +52,7 @@ test_enable = {{ default .Env.ODOO_TEST_ENABLE "False" }}
 test_file = {{ default .Env.ODOO_TEST_FILE "" }}
 test_tags = {{ default .Env.ODOO_TEST_TAGS "None" }}
 translate_modules = {{ default .Env.ODOO_TRANSLATE_MODULES "['all']" }}
+transient_age_limit = {{ default .Env.ODOO_TRANSIENT_AGE_LIMIT "1.0" }}
```
Comment (max3903) [inline odoo/12.0/templates/odoo.conf.tmpl:55]: @JevinD That is not an existing option in 12.0.

Comment (max3903) [inline odoo/12.0/templates/odoo.conf.tmpl:55]:
```suggestion
```

File: odoo/12.0/templates/odoo.conf.tmpl [hunk 3]
```diff
@@ -34,15 +34,14 @@ log_level = {{ default .Env.ODOO_LOG_LEVEL "info" }}
 logrotate = {{ default .Env.ODOO_LOGROTATE "False" }}
 longpolling_port = 8072
 max_cron_threads = {{ default .Env.ODOO_MAX_CRON_THREADS "1" }}
```
Comment (max3903) [inline odoo/12.0/templates/odoo.conf.tmpl:36]:
```suggestion
max_cron_threads = {{ default .Env.ODOO_MAX_CRON_THREADS "1" }}
osv_memory_age_limit = {{ default .Env.ODOO_OSV_MEMORY_AGE_LIMIT "1.0" }}
```

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
@JevinD Can I merge it?

**Comment by JevinD** [comment]
@max3903 Got held up on another ticket. I did not prep osi-erp/defiance/strymon yet.

File: odoo/12.0/entrypoint.sh [hunk 4]
```diff
@@ -94,13 +96,13 @@ function create() {
...
     echo "Creating $1"
-    createdb $1
+    createdb --maintenance-db=$DEFAULTDB $1
   fi
 }
 
 function drop() {
   echo "Dropping $1"
-  dropdb --if-exists $1
+  dropdb --if-exists --maintenance-db=$DEFAULTDB $1
```
Comment (max3903) [inline odoo/12.0/entrypoint.sh:105]: @JevinD Not sure that option exists on 12.0

**Review by max3903 — APPROVED** [review]

**Pull request merged**